src/Hazelcast.Net/Models/IndexOptions.cs

```diff
@@ -87,10 +87,12 @@ public IndexOptions AddAttribute(string attribute)
         /// <returns>This instance for chaining.</returns>
         public IndexOptions AddAttributes(params string[] attributes)
         {
+#pragma warning disable CA1062 // Validate arguments of public methods
```

Comment:
this is a public method, better actually validate the arguments with
```
if (attributes is null) throw new ArgumentNullException(nameof(attributes));
```
and while we're at it, there should be the same line in the constructor that accepts an `IEnumerable<string>` - no idea why it does not trigger a warning.

Reply:
I thought throwing could be breaking change since it's public but I'll add the check. ... I couldn't find anything meaninfull for the ctor case.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/682/commits/146cc487e6723800e0609781507b11b70e1c1c41

Reply:
ok